README.md

```diff
@@ -128,6 +128,69 @@ const metrics = new NewRelic({
+| `durationBlocked` | number | time until a socket was available in milliseconds | `0.587596` |
+| `durationDNS` | number | duration of DNS resolution in milliseconds | `0.441319` |
+| `durationConnect` | number | TCP connection duration in milliseconds | `1.138568` |
+| `durationSSL` | number | SSL handshake duration in milliseconds (https only) | `3.639361` |
+| `durationSend` | number | time it took to send the HTTP request. Note this currently does not work for streaming requests as done for our rendition uploads until we upgrade to Node 12+ in Adobe I/O Runtime. Until then the send time is included in the `durationConnect`. | `0.065885` |
+| `durationWait` | number | time between request was sent and first byte of response was received in milliseconds | `288.649511` |
+| `durationReceive` | number | time it took to receive the entire response body in milliseconds | `0.029158` |
+| `error` | string | Only set if there was a low-level connection error. Set as `true` in json, represented as `1` in NewRelic. | `1` |
+| `errorCode` | string | OS or nodejs error code (name or number) in case there was a low-level connection error. `110` means `ETIMEDOUT`. | `"ECONNRESET"` or `"110"` |
+| `errorMessage` | string | Error message in case there was a low-level connection error. | `"socket hang up"` |
+
+### Standard
+
+Sent for all metrics.
+
+| Attribute         | Format           | Description                     |
+|-------------------|------------------|---------------------------------|
+| `eventType`       | string           | Event type, required, [standard New Relic Insights type](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/insights-custom-data-requirements-limits). |
+| `timestamp`       | utc millis (?)   | The UTC timestamp to associate with the event. [Standard New Relic Insights type](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/insights-custom-data-requirements-limits). |
+| `requestId`       | string           | x-request-id of the original incoming http request |
+| `namespace`       | string           | OpenWhisk namespace of the action that sent the event. |
+| `package`         | string           | OpenWhisk package name of the action that sent the event.  |
+| `actionName`      | string           | OpenWhisk action name (without package) of the action that sent the event. |
+| `activationId`    | string           | OpenWhisk activation id of the action that sent the event. |
+| `orgId`           | string           | IMS organization id of the invoking technical account/integration/client. |
+| `appName`         | string           | Name of the invoking IO console integration (API key label) |
+| `clientId`        | string           | API key = IMS client id of the invoking technical account/integration/client. |
+| `sourceName`      | string           | Filename of the source. |
+| `sourceSize`      | number           | Size in bytes of the source. |
```

Comment:
`orgId` to `sourceSize` can be removed as they are not set by this library, they are custom Nui specific and set in `asset-compute-commons`.